### ISSUE TITLE
Linux Mint 18.1/Ubuntu 16.04 support -> Build Error

### ISSUE THREAD

**Issue by MartinX3** [moneymanagerex/moneymanagerex#1127]
make sure these boxed are checked before submitting your issue - thank you!
- Check OS
  - [ ] Windows
  - [ ] Mac OSX
  - [X] Linux Mint 18.1 alias Ubuntu 16.04
-  Check MMEX version
  - [X] 1.3.x
  - [ ] 1.2.x
  - [ ] 1.1 or older

Following INSTALL.Ubuntu.md

While running make package

CMakeFiles/mmex.dir/src/billsdepositspanel.cpp.o:billsdepositspanel.cpp:vtable for billsDepositsListCtrl: error: undefined reference to 'non-virtual thunk to wxG..enericListCtrl::GetSizeAvailableForScrollTarget(wxSize const&)'
CMakeFiles/mmex.dir/src/budgetingpanel.cpp.o:budgetingpanel.cpp:vtable for budgetingListCtrl: error: undefined reference to 'non-virtual thunk to wxGenericListCtrl::GetSizeAvailableForScrollTarget(wxSize const&)'
CMakeFiles/mmex.dir/src/general_report_manager.cpp.o:general_report_manager.cpp:vtable for sqlListCtrl: error: undefined reference to 'non-virtual thunk to wxGenericListCtrl::GetSizeAvailableForScrollTarget(wxSize const&)'

../../bin/lib/libLUA.a(loslib.c.o):loslib.c:function os_tmpname: warning: the use of `tmpnam' is dangerous, better use `mkstemp'
collect2: Fehler: ld gab 1 als Ende-Status zurück
CMakeFiles/mmex.dir/build.make:3008: die Regel für Ziel „../../bin/mmex“ scheiterte
make[2]: *** [../../bin/mmex] Fehler 1
CMakeFiles/Makefile2:68: die Regel für Ziel „CMakeFiles/mmex.dir/all“ scheiterte
make[1]: *** [CMakeFiles/mmex.dir/all] Fehler 2
Makefile:149: die Regel für Ziel „all“ scheiterte
make: *** [all] Fehler 2iel „CMakeFiles/mmex.dir/all“ scheiterte
make[1]: *** [CMakeFiles/mmex.dir/all] Fehler 2
Makefile:149: die Regel für Ziel „all“ scheiterte
make: *** [all] Fehler 2

Edit29.04.2017:
Same Error
[https://pastebin.com/zyQdSB1j](https://pastebin.com/zyQdSB1j)

**Comment by tomasfrtala** [comment]
I double checked it and I still can not reproduce your error. In my fresh installation of Linux Mint 18.1 I followed [instructions](https://github.com/moneymanagerex/moneymanagerex/blob/master/doc/INSTALL.Ubuntu.md#ubuntu-1604) and it finished with success. The same applies for Ubuntu 16.04 here you can find [log](https://pastebin.com/PevQMdx1)
![linuxmint-181-success](https://cloud.githubusercontent.com/assets/10168185/25559039/92295f28-2d33-11e7-9d5e-67044b8a40a7.png)

**Comment by tomasfrtala** [comment]
After further discussion we discovered these pitfalls of building mmex:
- Sources files were too old (_in this case it was solved by checkouting from latest master_) 
- Linkage problem (_this was not solved satisfactorily. Instead of building mmex, we decided to install last stable realse mmex-1.3.3.deb package generated from CI. And it worked_)

Closing due to inability of reproducing customer production environments settings.
